backend/lightning/lightning.go

```diff
@@ -154,13 +167,51 @@ func (lightning *Lightning) Deactivate() error {
 	return nil
 }
 
+// AssertActive returns an error if the lightning service not has been activated.
+func (lightning *Lightning) AssertActive() error {
```

Comment:
assert is the wrong word, as assert in all contexts I know (in Python, C, ...), it crashes/panics/exists.

`CheckActive` or `ValidateActive` would be better.